_posts/jobs/2016-09-30-android.md

```diff
@@ -38,3 +38,5 @@ You are a Software Engineer by heart. You have clean code for breakfast sprinkle
 We expect developing is not new to you. We will ask you to show your skills in these areas. We will also ask you to solve some problems for us to demonstrate your expertise.
 
 You may be young or you may be a veteran in software development. This is not the most important criteria but remember, this is senior position, the bar is set high.
+
+If you think you are this person - [get in touch with us](mailto:jobs@updudes.net)
```

Comment:
How about: 
> If this is you, get in touch with us! 


Then rebase and make these changes in all the job posts, please.